code-security/admin_guide/scan-monitor/monitor-fix-issues-in-scan/fix-issues-in-a-scan-result.adoc

```diff
@@ -0,0 +1,72 @@
+:topic_type: task
+
+[.task]
+
+== Fix Issues in a Scan Result
+
+On *Projects*, you can remediate scan results across all code categories by adding issues to the fix cart to create a PR with a suggested fix. For every issue found on Prisma Cloud console, you can view information like origin of the issue in a file or repository, policy violation, and suggestions to remediate the issue.
+
+[.procedure]
+
+. Access scan results on *Projects*.
+
+.. Select a code category with an issue.
+
+.. Select an issue from the resource block to view more information and suggested fixes in the resource explorer.
++
+In this example, on IaC Misconfiguration you see `aws_s3_bucket.public_read` resource block with an issue that has a fix.
++
+image::proj-22.png[width=800]
+
+. Create a PR with a fix using the suggestions in the resource explorer.
+
+.. Select an issue to see a fix suggestion in the resource explorer.
++
+You can fix one more issues at once by selecting issues across multiple resources or policy blocks and adding it to the fix cart.
+
+.. Select *FIX* to add the issue with a fix to the fix cart.
+
+.. Select *Submit* to create a PR with an issue fix.
++
+In this example, you see one or more issues added to the fix cart to from IaC Misconfiguration `google_container_cluster.workload_cluster` resource block.
++
+image::projects-fix.gif[width=800]
++
+To resolve an issue, you need to access the PR on the VCS console and merge the fix with the default branch.
++
+If the issue does not have a fix suggestion, you can choose a *Manual Fix* or *Suppress*.
+
+=== Manual Fix an issue
+
+You can perform a manual fix for all issues. A manual fix enables you to access a specific commit to review the code to then resolve the issue manually using the policy guidelines on the Prisma Cloud console.
+
+image::proj-24.png[width=800]
+
+[.task]
+
+=== Suppress issues in a scan result
+
+On *Code Security > Projects*, add a suppression rule to suppress issues across views to mitigate scan results.
+
+[.procedure]
+
+. Access a code category then select the issue in the resource block to view more information and suggested fixes in the resource explorer.
+
+. Select *Suppress* and then enter relevant information as *Justification*.
++
+image::suppress.png[width=600]
++
+You can optionally add an *Expiration Time* for the suppression.
+
+. Select *Suppress by* to suppress issues based on the two suppression types.
++
+image::suppress-type.png[width=400]
++
+* *Resource*: This enables you to suppress the issues by resources and at your next scan these resources will not be scanned. You can also view the number of resources that will be affected to make an informed decision.
+* *Repositories*: This enables you to selectively suppress the issues across repositories. You are required to select the repositories from the list on the console.
```

Comment:
@anaghapamidi we do support more than this two suppressions. is there anything in our docs mentioning it?
if not - can we just refer it as an example for now and extend this content later on?

Reply:
@GiladMark - Please have a look here. Have added the rest.